install/debian/InstallHalyard.sh

```diff
@@ -181,23 +103,25 @@ if [ "\$yes" != "y" ] && [ "\$yes" != "Y" ]; then
   exit 0
 fi
 
+apt purge -y spinnaker-halyard
+
 rm /opt/halyard -rf
 rm /var/log/spinnaker/halyard -rf
 rm -f /usr/local/bin/hal /usr/local/bin/update-halyard
 
 echo "Deleting halconfig and artifacts"
 rm /opt/spinnaker/config/halyard* -rf
-rm $halconfig_dir -rf
```

Comment:
Don't we still need this, or at least the equivalent without a $halfconfig_dir variable?  Seems tempting to leave that variable around...